### ISSUE TITLE
Inject Key Storage Password into Option's ValuesUrl

### ISSUE THREAD

**Issue by molinacarlos** [rundeck/rundeck#8019]
Actually cannot use KeyStorage information into Option's Value URL. In [this stackoverflow question](https://stackoverflow.com/questions/74351618/inject-key-storage-password-into-options-valuesurl) have detailed information about it.  This feature could lead to make big improvements in cascading remote options, to get information through a remote API, regardless security issues that could be raised for a non experienced user (sensitive information is presented on Failed message) exposing secret data.

![image](https://user-images.githubusercontent.com/20499477/203521122-244b390b-9824-46db-8a5e-87ba36d1f520.png)

In summary, cannot use key storage information in a ${option.option_name.value}. The only way to use secure information, is by text option. As a result of possible safety risks, when sensitive information is exposed

A clear and concise description of what the business problem is.  Ex. I'm always frustrated when [...] / I am trying to [...]

**Describe the solution you'd like**
New option value or new option type call, that abe rundeck to "preprocess" this kind of options before the executions. May be, using some AJAX on the user interface to retrive secure options data (key storage)

**Comment by pgmgb** [comment]
@molinacarlos there is already an issue about that open since more than one year.
https://github.com/rundeck/rundeck/issues/7091

**Comment by SergeyNosko** [comment]
I would bump this up

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
